### PR TITLE
Get rid of overflow

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/functionlib.py [hunk 1]
```diff
@@ -238,7 +238,10 @@ def function_from_transform(self, x: np.ndarray) -> float:
         results = []
         for block in x:
             results.append(self._func(block))
-        return float(self._aggregator(results))
+        try:
+            return float(self._aggregator(results))
+        except OverflowError:
+            return 1.0e20
```
Comment (jrapin) [inline nevergrad/functions/functionlib.py:244]: how about float("inf") instead, this way optimizser deal with it the way they want.

Comment (teytaud) [reply nevergrad/functions/functionlib.py:244]: A bit zfraid that some optimization methods will crash. Max-float might also be risky.

Comment (jrapin) [reply nevergrad/functions/functionlib.py:244]: https://github.com/facebookresearch/nevergrad/blob/93fd71736be88eec272188436919aa0d06cfbf11/nevergrad/optimization/base.py#L323-L331

It should not

Comment (teytaud) [reply nevergrad/functions/functionlib.py]: You're right. Modified accordingly.

**Pull request merged**